.Rproj.user/31B863D3/sources/s-31C9B9F0/BE3B08A2-contents

```diff
@@ -0,0 +1,27 @@
+library("project.margarida")
+
+context("Core Translate Functionality")
+
+test_that("translate() accepts string input only", {
+  expect_error(translate(34,3))
+  expect_error(translate(TRUE,FALSE))
+})
+
+test_that("translate() returns a different string from input text", {
+  # variables don't exist yet for this test 
+  # expect_false(isTRUE(all.equal(text, output_text)))
+})
+
+test_that("translate() accepts only available language codes as second argument", {
```

Comment:
Good tests that capture the core functionality! I am wondering how the function would react if you gave it gibberish or grammatically incorrect language? Would there be a way to test for that?